UpskillWikipediaUITests/XCUITestHelpers/BaseTestCase.swift

```diff
@@ -0,0 +1,30 @@
+import XCTest
+
+class BaseTestCase: XCTestCase {
+
+    var app: XCUIApplication!
+    private lazy var skipButton = app.staticTexts["Skip"]
```

Comment:
This should be part of the page object for Initial app tour views, not part of this class.